d3a_interface/sim_results/__init__.py

```diff
@@ -50,6 +50,17 @@ def is_buffer_node_type(area):
     return area['type'] == "InfiniteBusStrategy"
 
 
+def is_house_node_type(area):
+    return get_unified_area_type(area) == "Area"
+
+
+def has_no_grand_children(area):
```

Comment:
```suggestion
def has_no_grandchildren(area):
```
(or see comment below)